src/domains/idpay-app/api/idpay_onboarding_workflow/openapi.onboarding.yml.tpl

```diff
@@ -0,0 +1,404 @@
+openapi: 3.0.1
+info:
+  title: IDPAY Onboarding Workflow IO API
+  description: IDPAY Onboarding Workflow IO
+  version: '1.0'
+servers:
+ - url: https://api-io.dev.cstar.pagopa.it/idpay/onboarding
+paths:
+  /citizen:
```

Comment:
Cambiare con /onboarding

Reply:
@GiovanniMancini @dariopelliccioli 
Can we update this as agreed in the last call?

Reply:
Changed with commit -> https://github.com/pagopa/cstar-infrastructure/pull/452/commits/93f826c0b6f5e791eb24269cbe934e080540af13